extensions/interactions/NumberWithUnits/directives/oppia-interactive-number-with-units.directive.ts

```diff
@@ -91,8 +91,6 @@ angular.module('oppia').directive('oppiaInteractiveNumberWithUnits', [
           ctrl.$onInit = function() {
             $scope.$watch('$ctrl.answer', function(newValue) {
               try {
-                var numberWithUnits =
-                  NumberWithUnitsObjectFactory.fromRawInputString(newValue);
```

Comment:
Oh no, this is required. The errors thrown in `fromRawInputString()` function is caught by the try catch statement. Maybe the variable is not used but the function absolutely is.
Please make sure you verify all changes you do and make sure they are actually not used at all.

Reply:
Readded it after removing the variable.

Reply:
Please address the entire comment, you did not mention whether you verified all changes.

If you did not, then this PR will be inevitably rolled back. Please save the trouble for future testers, and yourself, by diligently ensuring none of this is causing breaking changes. @aks681's review suggests that this would have broken develop, so please make sure it does not.

Reply:
@brianlinUM @aks681 How should I test the changes?

Reply:
You can create a interaction with number with input interaction. Then open the learner view. If you type invalid input like 10m///s instead of 10m/s (basically units that don't exist), then an error should be shown below the text box.

Reply:
@aks681 I've reverted the changes and tested. The error message is there as expected.